Dockerfile

```
@@ -13,46 +13,307 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
+# WARNING: THIS DOCKERFILE IS NOT INTENDED FOR PRODUCTION USE OR DEPLOYMENT.
+#
+# Arguments of the build
+ARG PYTHON_BASE_IMAGE="python:3.6-slim"
+ARG AIRFLOW_VERSION="2.0.0.dev0"
+# Which image is used as dependency for the main image
+ARG APT_DEPS_IMAGE="airflow-apt-deps"
+# Speeds up building the image - cassandra driver without CYTHON saves around 10 minutes
+ARG CASS_DRIVER_NO_CYTHON="1"
+# Build cassandra driver on multiple CPUs
+ARG CASS_DRIVER_BUILD_CONCURRENCY="8"
+# By default PIP install is run without cache to make image smaller
+ARG PIP_CACHE_DIRECTIVE="--no-cache-dir"
+# Additional python deps to install
+ARG ADDITIONAL_PYTHON_DEPS=""
+# PIP version used to install dependencies
+ARG PIP_VERSION="19.0.1"
+# By increasing this number we can do force build of all dependencies
+ARG FORCE_REINSTALL_ALL_DEPENDENCIES="1"
```

I know exactly what this is doing but people less familar with Docker might see `FORCE_REINSTALL_ALL_DEPENDENCIES="1"` and think `1` = enable this feature.

How about?

```suggestion
ARG DEPENDENCIES_EPOCH_NUMBER="1"
```

I've chose "epoch" here after the concept from debian version number (2:1.0-1, `2:` is the epoch)

---

I like it :). Will change.

---

Done.